tests/.suitespec.json

```diff
@@ -768,7 +768,7 @@
             "@contrib",
             "@tracing",
             "@dbapi",
-            "tests/contrib/shared_tests.py"
```

Comment:
we need both shared_tests and shared_tests_async, right?

Reply:
this is dbapi_async's test suite. the sync version is used for sync dbapi tests (we really should get some way to replicate sync/async code and deduplicate this stuff)